packages/elements/src/components/ino-snackbar/ino-snackbar.tsx

```diff
@@ -67,6 +73,14 @@ export class Snackbar implements ComponentInterface {
       this.handleSnackbarHide(e)
     );
     this.configureTimeout();
+    if (this.inoStayVisibleOnHover) {
+      this.snackbarElement.addEventListener('mouseenter', () => {
+        if (this.timeout) clearTimeout(this.timeout);
```

Comment:
We can shorten this if we extract this to its own function. e.g.:
```ts
componentDidLoad() {
	// ...
	this.snackbarElement.addEventListener('mouseenter', this.interruptTimeout);
	// ...
}

disconnectedCallback() {
	// ...
	this.snackbarElement.removeEventListener('mouseenter', interruptTimeout);
	// ...
}

interruptTimeout(): void {
	if (this.timeout) clearTimeout(this.timeout);
}
```

Reply:
Good suggestion! I applied it